scripts/auth-open.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+import json
+import os
+import base64
+import requests
+
+requests.packages.urllib3.disable_warnings()
+
+#
+# From : https://github.com/dcos/dcos/blob/master/test_util/helpers.py#L35
```

Comment:
@orlandohohmeier well, I guess if we move it inside the `test_util` we can do an import from it.

@mellenburg since the script was executed from within the `config` directory, we had to do the `sys.path.append` hack to import the `test_util` directory, so we thought it was cleaner to just duplicate the token.